src/app.ts

```diff
@@ -272,28 +201,24 @@ if (process.env.MANUALLY_START_QUEUE === 'true') {
 }
 
 // Initialize services that need it
-const fediverseBridge = container.resolve<FediverseBridge>('fediverseBridge');
-fediverseBridge.init();
+container.resolve<FediverseBridge>('fediverseBridge').init();
```

Comment:
Would be great to get these `init`s handled by the container/registration for us eventually